extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -1014,7 +1004,7 @@ class QueueProcessor extends EventEmitter {
      * @param {Logger} log - Logger
      * @returns {string} Error response string or undefined
      */
-    handleMetrics(res, log) {
+    static handleMetrics(res, log) {
```

Comment:
I don't think this needs to be static and am wondering why you changed it

Reply:
since we create multiple instances of queue populator in the same process, it needs to be static indeed.

I could just move it to task.js, but I kept it here for consistency with the other "tasks".

Reply:
> since we create multiple instances of queue populator in the same process

we do? I thought we just create the one in `bin/queuePopulator.js` and pass it around everywhere. That was an assumption made when building the metric handlers.

Reply:
this is not QueuePopulator, but QueueProcessor : which has one instance (and consumer group) for each replicaiton site.
In S3C we used to run multiple node processes for these, but in Zenko they all got moved into the same node process.

Reply:
Wouldn't this mean we don't need this to be static then?

Reply:
It needs to be static since it called from task.js, in the /metrics route callback while there are multiple instances of QueueProcessor class (one per site)

Reply:
oh right in 8x we have multiple processors. I guess making it a static function that just calls the prom stuff is fine. Personally I think a better solution would be to move this to an exported function or a function under the metric handler but that would break the single export we have.

Reply:
I agree it is not very nice, and quite redundant since we have this in a few places. But I will leave this cleanup for another time...